### ISSUE TITLE
Object fb-checkpresence.0.AppleWanDominik is invalid: obj.type has to exist.

### ISSUE THREAD

**Issue by domvo** [afuerhoff/ioBroker.fb-checkpresence#30]
The [history adapter](https://github.com/ioBroker/ioBroker.history) reports in the following in the logs:

```
Object fb-checkpresence.0.<deviceName> is invalid: obj.type has to exist.
This object will not be created in future versions. Please report this to the developer.
```

**Comment by afuerhoff** [comment]
Please delete the object manually and restart the adapter.

**Comment by afuerhoff** [comment]
Hallo, die Testversion 1.1.0 kann von Github geladen werden.
Weitere Infos hier: https://forum.iobroker.net/topic/37959/test-adapter-fb-checkpresence-v1-1-x

**Comment by afuerhoff** [comment]
Hello, have you had any problems in the meantime with the same error?

**Comment by afuerhoff** [comment]
Fixed with 1.1.4.
